incubator/keycloak/values.yaml

```diff
@@ -0,0 +1,105 @@
+image:
+  ## Keycloak image.
+  ##
+  repository: jboss/keycloak
+
+  ## keycloak image version.
+  ## ref: https://hub.docker.com/r/jboss/keycloak/tags/
+  ##
+  tag: 3.3.0.Final
+
+  ## Specify a imagePullPolicy: 'Always' if imageTag is 'latest', else set to 'IfNotPresent'.
+  ## ref: https://kubernetes.io/docs/user-guide/images/#pre-pulling-image
+  ##
+  pullPolicy: IfNotPresent
+
+keycloak:
+  ## First keycloak username
+  ## This user has full admin access
+  username: keycloak
+
+  ## First keycloak password
+  password: keycloak
+
+  ## Keycloak logging level
+  loglevel: INFO
+
+  ## Configure resource requests and limits.
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources:
+    # limits:
+    #   cpu: "100m"
+    #   memory: "1024Mi"
+    requests:
```

Comment:
Don't specify defaults for resource. We generally recommend to leave this a conscious choice of the user.